ci/dev/vscode.patch

```diff
@@ -3859,16 +3902,13 @@ index 021af6e0f8983c492f9cdd048ba2dcae7640bc1d..4474a93beba03365709c3cda98b68213
  		module = module.with({ path: ensureSuffix(module.path, '.js') });
  		const response = await fetch(module.toString(true));
 diff --git a/src/vs/workbench/browser/actions/navigationActions.ts b/src/vs/workbench/browser/actions/navigationActions.ts
-index 7344a3a29b32f7b370b99bf0cfdc79a322195ff8..dc21396e83e2f53914447d3460c2ee1103ecb28e 100644
+index 7344a3a29b32f7b370b99bf0cfdc79a322195ff8..46c900da7610d9ca6ddff4556b5d93b8dd58588c 100644
 --- a/src/vs/workbench/browser/actions/navigationActions.ts
 +++ b/src/vs/workbench/browser/actions/navigationActions.ts
-@@ -310,4 +310,8 @@ actionsRegistry.registerWorkbenchAction(SyncActionDescriptor.from(FocusNextPart,
- actionsRegistry.registerWorkbenchAction(SyncActionDescriptor.from(FocusPreviousPart, { primary: KeyMod.Shift | KeyCode.F6 }), 'View: Focus Previous Part', CATEGORIES.View.value);
+@@ -311,3 +311,6 @@ actionsRegistry.registerWorkbenchAction(SyncActionDescriptor.from(FocusPreviousP
  
  const workbenchRegistry = Registry.as<IWorkbenchContributionsRegistry>(WorkbenchExtensions.Workbench);
--workbenchRegistry.registerWorkbenchContribution(GoHomeContributor, LifecyclePhase.Ready);
-+// See https://github.com/cdr/code-server/issues/2328
-+// workbenchRegistry.registerWorkbenchContribution(GoHomeContributor, LifecyclePhase.Ready);
+ workbenchRegistry.registerWorkbenchContribution(GoHomeContributor, LifecyclePhase.Ready);
 +export const _1 = workbenchRegistry;
 +export const _2 = GoHomeContributor;
 +export const _3 = LifecyclePhase.Ready;
```

Comment:
Can these be removed as well? I don't think they're in VS Code's original source.

Reply:
Ah that's true, these were part of my previous patch @jsjoeio

Reply:
Yeah, I'm not where those came from. Removing!

Reply:
I'm still having issues with the `yarn.lock` file. I will open a separate PR to address that!